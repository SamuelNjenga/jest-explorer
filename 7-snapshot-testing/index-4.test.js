//Any given value that is not a matcher will be checked exactly and saved to the snapshot:
it('will check the values and pass', () => {
    Date.now = jest.fn(() => 1482363367071);
    const user = {
        createdAt: Date.now(),
        name: 'Computer Science... Software Engineering',
        school: 'The Great School Of Life'
    };

    expect(user).toMatchSnapshot({
       // createdAt: expect.any(Date),
        name: 'Computer Science... Software Engineering'
    });
});