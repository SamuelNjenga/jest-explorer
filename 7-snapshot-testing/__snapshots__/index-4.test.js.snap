// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`will check the values and pass 1`] = `
Object {
  "createdAt": 1482363367071,
  "name": "Computer Science... Software Engineering",
  "school": "The Great School Of Life",
}
`;
